TeamCode/src/main/java/suitbots/MecanumTeleop.java

```diff
@@ -0,0 +1,83 @@
+package suitbots;
+
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import com.qualcomm.robotcore.eventloop.SyncdDevice;
+
+
+/**
+ * Created by Samantha on 9/17/2017.
+ */
+
+@TeleOp(name = "Issac 7", group = "Tournament")
+public class MecanumTeleop extends OpMode {
+    private Robot robot = null;
+    private Controller g1, g2;
+    private boolean debug_mode = false;
+
+
+    @Override
+    public void init() {
+        robot = new Robot(hardwareMap);
+
+        g1 = new Controller(gamepad1);
+        g2 = new Controller(gamepad2);
+    }
+
+    @Override
+    public void init_loop() {
+        g1.update();
+        if (g1.AOnce()) {
+            debug_mode = ! debug_mode;
+        }
+        telemetry.addData("Debug? (a)", debug_mode ? "on" : "off");
+        telemetry.addData("Ready?", "YES.");
+        telemetry.update();
+    }
+
+    @Override
+    public void start() {
+        robot.onStart();
+    }
+
+    @Override
+    public void stop() {
+        robot.onStop();
+    }
+
+
+//    private void g2Loop(Controller g) {
+//        if (g.X()) {
+//            robot.setBackPower(-1.0);
+//        } else if (g.Y()) {
+//            robot.setBackPower(1.0);
+//        } else {
+//            robot.setBackPower(0.0);
+//        }
+//
+//        if (g.A()) {
+//            robot.setFrontPower(-1.0);
+//        } else if (g.B()) {
+//            robot.setFrontPower(1.0);
+//        } else {
+//            robot.setFrontPower(0.0);
+//        }
+//
+//
+//
+//    }
+
+    @Override
+    public void loop() {
+        g1.update();
+        g2.update();
+//        g1Loop(g1);
```

Comment:
I want to say that g1Loop and g2Loop were important :) Here's [some example Mecanum code](https://github.com/cporter/ftc_app/blob/rr/pre-season/TeamCode/src/main/java/soupbox/Mecanum.java) that I've been posting on Reddit if you'd like an example.

If you really want to make this easy, add a method to `Robot` that takes rotation, direction and speed and translates *that* into motor powers.